medusa/providers/torrent/json/hdbits.py

```diff
@@ -44,48 +44,64 @@ def __init__(self):
         self.url = 'https://hdbits.org'
         self.urls = {
             'search': urljoin(self.url, '/api/torrents'),
-            'rss': urljoin(self.url, '/api/torrents'),
-            'download': urljoin(self.url, '/download.php'),
+            'download': urljoin(self.url, '/download.php?{0}'),
         }
 
         # Proper Strings
+        self.proper_strings = []
 
         # Miscellaneous Options
 
         # Torrent Stats
+        self.minseed = None
+        self.minleech = None
 
         # Cache
-        self.cache = HDBitsCache(self, min_time=15)  # only poll HDBits every 15 minutes max
+        self.cache = tv.Cache(self, min_time=10)  # only poll HDBits every 10 minutes max
 
     def search(self, search_strings, age=0, ep_obj=None):
         """
         Search a provider and parse the results.
-
         :param search_strings: A dict with mode (key) and the search value (value)
         :param age: Not used
         :param ep_obj: Not used
         :returns: A list of search results (structure)
         """
-        # FIXME
+
         results = []
 
-        logger.log('Search string: {0}'.format(search_strings), logger.DEBUG)
+        logger.log('Search strings: {0}'.format(search_strings), logger.DEBUG)
```

Comment:
Pls use new style logging. You can check one of labrys 369 open PR's for examples.